### PR TITLE
Updated Quiz Questions and Removed Incomplete Section

### PR DESCRIPTION
**Pull request by dustin-miao**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by dustin-miao** [comment]
Addresses #3375

File: content/2_Bronze/Complete_Rec.mdx [hunk 1]
```diff
@@ -1134,21 +1086,21 @@ make code consistent, improve some of these explanations
...
 	    <Quiz.Answer>
 	      Recursion often runs into the issue of stack size, which can cause unexpected runtime errors
 	      <Quiz.Explanation>
-		Incorrect: A hallmark of problems featuring complete search with permutations/subsets is that $n$ is small, usually $\leq 20$. This means that recursion won't go particularly deep, and there won't be any issues with stack size. 
+		Incorrect: Most complete search with permutations/subsets is that $n$ is small, usually $n\leq 20$. This means that recursion won't go particularly deep, and there won't be any issues with stack size. 
 	      </Quiz.Explanation>
 	    </Quiz.Answer>
 	    <Quiz.Answer correct>
-	      Iterative methods are easier to implement and are often faster and less buggy. 
+	      Using the C++ library function is easier and less buggy.
```
Comment (SansPapyrus683) [inline content/2_Bronze/Complete_Rec.mdx:1097]: shouldn't these questions be language agnostic?

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
wow, dustin actually doing something?

File: content/2_Bronze/Complete_Rec.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply content/2_Bronze/Complete_Rec.mdx:1097]: at this point i'd rather just delete the question

File: content/2_Bronze/Complete_Rec.mdx [hunk 2]
```diff
@@ -1134,21 +1086,21 @@ make code consistent, improve some of these explanations
 	    </Quiz.Answer>
 	  </Quiz.Question>
 	  <Quiz.Question>
-	    Why are iterative methods of generating permutations and subsets like $\texttt{next\_permutation}$ and bitmasking preferable over recursive implementations?
+	    Why is using `std::next_permutation` generally preferable to writing your own recursive function?
```
Comment (bqi343) [inline content/2_Bronze/Complete_Rec.mdx:1089]:
```suggestion
	    In C++, why is using `std::next_permutation` generally preferable to writing your own recursive function to iterate through permutations?
```

**Pull request merged**